src/modules/cookies/index.tsx

```diff
@@ -29,5 +30,6 @@ export function getGlobalCookies(req?: {
       language: req?.cookies[LANGUAGE_COOKIE_NAME] ?? null,
     },
     headersAcceptLanguage: req?.headers['accept-language'] ?? '',
+    referer: req?.headers.referer ?? '',
```

Comment:
Do we need to set it as a cookie?

Reply:
I might be reading the code wrong here, but this shouldn't set it as a cookie. A bit weird naming of the function maybe. Should probably be "getInitialData" or something

Reply:
Nevermind me, they aren't set as a cookie, only read from cookie of course. Confused by the type and naming of function as well.